BridgeAppSDK/SBAProfileItem.swift

```diff
@@ -523,7 +557,170 @@ open class SBAUserDefaultsProfileItem: SBAProfileItemBase {
 
 }
 
-open class SBAFullNameProfileItem: SBAKeychainProfileItem, SBANameDataSource {
+enum SBAProfileParticipantSourceKey: String {
+    case firstName
+    case lastName
+    case email
+    case externalId
+    case notifyByEmail
+    case sharingScope
+    case dataGroups
+}
```

Comment:
Are these defined here rather than as obj-c enum strings b/c you explicitly want them *not* to extend?